lib/features/mailbox_dashboard/presentation/mailbox_dashboard_controller.dart

```diff
@@ -65,13 +67,17 @@ class MailboxDashBoardController extends ReloadableController {
   final accountId = Rxn<AccountId>();
   final userProfile = Rxn<UserProfile>();
   final searchState = SearchState.initial().obs;
-  final suggestionSearch = <String>[].obs;
+  final shouldShowSuggestionDropdown = false.obs;
+  final recentSearchs = <RecentSearchHiveCache>[].obs;
+  final searchFilters = <SearchFilter>[].obs;
+  final searchTyping = ''.obs;
```

Comment:
Need use final searchTyping = RxString('');

Reply:
Changed to RxString